rudder-webapp/SOURCES/rudder-upgrade

```diff
@@ -296,6 +297,16 @@ upgrade_techniques() {
   fi
 }
 
+# Add indexes cn,softwareVersion,directiveId in slapd.conf
+SLAPD_CONF="/opt/rudder/etc/openldap/slapd.conf"
+for i in "directiveId" "softwareVersion" "cn"; do
+  line="index\s*${i}\s*eq"
+  grep -E "${line}" ${SLAPD_CONF} > /dev/null 2>&1 || (
+    echo "Adding LDAP index on attribute: ${i}"
+    sed -i "/index\s*objectClass\s*eq/a index\t${i}\teq" ${SLAPD_CONF}
```

Comment:
if indexes are added, a slapindex -q need to be done, when service rudder-slapd is stopped

Reply:
no idea where/how to do that